dataactcore/models/errorInterface.py

```diff
@@ -82,6 +82,9 @@ def checkNumberOfErrorsByJobId(self, jobId):
         queryResult = self.session.query(ErrorMetadata).filter(ErrorMetadata.job_id == jobId).all()
         numErrors = 0
         for result in queryResult:
+            if result.severity_id != valDb.getRuleSeverityId(errorType):
+                # Don't count non-fatal
```

Comment:
Very minor point, but it looks like this function is set up to be more flexible than this comment implies (e..g, just finding non-fatal validation results).